jenkins/Dockerfile

```diff
@@ -1,17 +1,13 @@
-FROM ubuntu:16.04
+FROM golang:1.10-alpine3.7
 
-RUN apt-get update
-RUN apt-get install -y curl software-properties-common git
-RUN curl -fsSL get.docker.com | sh
-
-##
-## Go
-##
-RUN add-apt-repository ppa:longsleep/golang-backports
-RUN apt-get update
-RUN apt-get install -y golang-go
-ENV GOPATH=$HOME/go
-ENV PATH=$PATH:$HOME/go/bin
+RUN apk add --no-cache curl
+RUN apk add --no-cache docker
+RUN apk add --no-cache git
+RUN apk add --no-cache sudo
+RUN apk add --no-cache bash
+RUN apk add --no-cache mongodb
+RUN apk add --no-cache make
+RUN apk add --no-cache gcc
```

Comment:
Please install the packages in one line, so that fewer layers(smaller size) are used.
```
RUN apk add --no-cache curl \
    docker \
    git \
    ...
```

Reply:
ＯＫ